modules/nf-core/sickle/main.nf

```diff
@@ -0,0 +1,65 @@
+process SICKLE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::sickle-trim=1.33"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/sickle-trim:1.33--h7132678_7':
+        'quay.io/biocontainers/sickle-trim:1.33--h5bf99c6_6' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("${prefix}.se.trimmed.fastq.gz"),        optional:true, emit: single_trimmed
+    tuple val(meta), path("${prefix}.pe{1,2}.trimmed.fastq.gz"),   optional:true, emit: paired_trimmed
+    tuple val(meta), path("${prefix}.singleton.trimmed.fastq.gz"), optional:true, emit: singleton_trimmed
+    tuple val(meta), path("*.log"), emit: log
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    def qual_type = "${meta.qual}"
+
+    if (meta.single_end){
+    """
+    sickle \\
+        se \\
+        $args \\
+        -f $reads \\
+        -t $qual_type \\
+        -o ${prefix}.se.trimmed.fastq.gz \\
+        -g \\
```

Comment:
Is the `-g` argument mandatory here?

Reply:
Nope, it is not mandatory argument but it gives compressed fastq output. I refer to couple of nf-core modules and noticed that it is preferred way to get output fastq file (notice "gzip" argument in here: https://github.com/nf-core/modules/blob/master/modules/nf-core/trimgalore/main.nf). However, I can remove it if it violates the standard templates of nf-core/modules. Please give your suggestion.

Reply:
No that's perfect! Well done :)